nuclio/build.py

```diff
@@ -84,9 +85,10 @@ def build_file(filename='', name='', handler='', archive=False, project='',
         code = add_kind_footer(kind, config, code)
 
     name = normalize_name(name or filebase)
+    normalized_filebase = normalize_name(filebase)
```

Comment:
why not resolve the full name here:
```suggestion
    # Avoid import issues if the filename is the same as an existing Python library by adding a suffix
    filebase_name = normalize_name(filebase) + '_nuclio'
```

Reply:
@TomerShor in the`normalize_name` we get rid of using all the `_` symbols, so I'd go then with adding `-nuclio` then